models/migrations/v51.go

```diff
@@ -0,0 +1 @@
+package migrations
```

Comment:
copyright head

Reply:
Done.

Reply:
OK apparantly it didn't update right away in the pr, but when wiewing it in my repo it is updated: https://github.com/kolaente/gitea/blob/master/models/migrations/v51.go